### DISCUSSION TITLE
Exact quadratic prediction by the Sibson interpolation method

### DISCUSSION THREAD

**Discussion by stla** [CGAL/cgal#7298] [Q&A]
Hello,

The doc claims: 

![sibson](https://user-images.githubusercontent.com/4466543/222725540-5e2cb244-b637-480d-8093-02a365071c08.png)

However I tried with a general quadratic form `a + bx*x + by*y + cxx*x^2 + cyy*y^2 + cxy*x*y` and I reproduced the exact values by interpolation. Are you sure of this restriction?

**Comment by stla** [comment]
Still checked and that's not true. It was a particular case.

**Comment by stla** [comment]
But seems ok when `cxy=0`.

**Comment by lrineau** [comment]
I doubt there are many CGAL developers familiar with interpolation methods. Can you please post your C++ code?

**Comment by stla** [comment]
My code is the same as the one in [this example](https://doc.cgal.org/latest/Interpolation/index.html#InterpolationExampleSibson). But maybe I still was in a particular case.
